app/src/test/java/protect/card_locker/LoyaltyCardCursorAdapterTest.java

```diff
@@ -162,9 +162,9 @@ public void TestCursorAdapterFontSizes() {
 
     @Test
     public void TestCursorAdapterStarring() {
```

Comment:
This test already checks `assertEquals(View.VISIBLE, star.getVisibility());`, maybe we can test `assertEquals(View.VISIBLE, archiveIcon.getVisibility());` here too so we have archive tests too (or whatever the archive icon is called, I forgot)